be1-go/cli/organizer/organizer.go

```diff
@@ -2,23 +2,13 @@ package organizer
 
 import (
 	"encoding/base64"
-	"fmt"
-	"log"
-	"net/http"
 	"student20_pop"
 	"student20_pop/hub"
 
-	"github.com/gorilla/websocket"
 	"github.com/urfave/cli/v2"
 	"golang.org/x/xerrors"
 )
 
-var upgrader = websocket.Upgrader{
-	ReadBufferSize:  1024,
-	WriteBufferSize: 1024,
-	CheckOrigin:     func(r *http.Request) bool { return true },
-}
-
```

Comment:
What's the reason for deleting these lines?

Reply:
I moved them to the file serveWs.go in the hub package along with the functions that use the upgrader. I did this to avoid duplicating code because these functions are identical for witness and organizer.